### ISSUE TITLE
[node] Missing `brotliCompressionSync` in module `zlib`

### ISSUE THREAD

**Issue by marvinhagemeister** [denoland/deno#19162]
This is used in `ts-node`. Came across this when trying to the `lint:style` script from the https://github.com/ant-design/ant-design project.

Node docs https://nodejs.org/api/zlib.html#zlibbrotlicompresssyncbuffer-options 

## Steps to reproduce

1. Create file `foo.js` with the following contents:
```js
import zlib from 'node:zlib';
import { Buffer } from 'node:buffer';

const res = zlib.brotliCompressSync(Buffer.from('foobar', 'utf-8')).toString('base64');
console.log(res);
```
2. Run `deno run -A foo.js`

```sh
error: Uncaught Error: Not implemented: brotliCompressSync
const res = zlib.brotliCompressSync(Buffer.from('foobar', 'utf-8')).toString('base64');
                 ^
    at notImplemented (ext:deno_node/_utils.ts:7:11)
    at Object.brotliCompressSync (ext:deno_node/zlib.ts:41:5)
    at file:///Users/marvinhagemeister/dev/github/ant-design/foo.mjs:4:18
```

**Comment by marvinhagemeister** [comment]
Related https://github.com/denoland/deno/issues/12716

**Comment by bartlomieju** [comment]
I think it's more related to https://github.com/denoland/deno/issues/19066; @levex could you fix that one as well?

**Comment by levex** [comment]
This is currently pending adding Brotli to the HTTP streaming compression and I expect that and the fix to #19066 (which should also make it easier to add to zlib) land this week. :-)

**Comment by bartlomieju** [comment]
This is now implemented :)
